dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -4205,7 +4205,6 @@ definitions:
     - "namespace"
     - "private_access"
     - "registry"
-    - "toolname"
```

Comment:
Is this intended to be gone?

Reply:
Yes it is no longer required since it is nullable. I made it the same as workflowname

Reply:
nvm, seems fine